cursor.js

```diff
@@ -50,6 +50,10 @@ WL.registerComponent('cursor', {
 
         this.globalTarget = this.object.addComponent('cursor-target');
 
+        if(this.cursorRayObject) {
```

Comment:
There is a `if(this.cursorRayObject)` futher below already, would be great if you could reuse that! :)